scripts/sdk_report.py

```
@@ -0,0 +1,588 @@
+    r"macos.*wheel",
+    r"manual updates",
+    r"chore\(",
+    r"update.*readme",
+    r"update.*changelog",
+    r"ci:",
+    r"refactor\(",
+    r"correct.*typo",
+]
+
+
+def parse_release_body(version: str, body: str) -> list[Feature]:
```

Ruff (0.15.6)

[warning] Too many branches (23 > 12)

(PLR0912)